src/TextInput/TextInput.svelte

```diff
@@ -83,6 +83,12 @@
    */
   export let ref = null;
 
+  /**
+   * Set to `true` to mark the field as required
+   * @type {boolean} [required="false"]
```

Comment:
`"false"` should be `false`

Reply:
Okay I will complete this change. 

Thank you